File: FizzBuzz.py
# -*-coding:utf-8-*-

for x in range(1,100):
	response = ''

	if (x % 3) == 0:
		response += 'Fizz'

	if (x % 5) == 0:
		response += 'Buzz'	

	if response == '':
		response = str(x)
		
	print(response)

File: fizzbuzz.erl
-module(fizzbuzz).
-export([print/0]).


print() -> forIn(lists:seq(1, 100)).


forIn([FirstItem|End]) -> print(FirstItem), forIn(End);
forIn([]) -> ok.

print(Item) when (Item rem 3 =:= 0) and (Item rem 5 =:= 0) -> io:format("FizzBuzz~n");
print(Item) when Item rem 3 =:= 0 -> io:format("Fizz~n");
print(Item) when Item rem 5 =:= 0 -> io:format("Buzz~n");
print(Item) -> io:format("~p~n", [Item]).
